package.json

```diff
@@ -12,9 +12,9 @@
         "litecoin",
         "scrypt"
     ],
-    "homepage": "https://github.com/s-nomp/node-stratum-pool",
+    "homepage": "https://github.com/HorizenOfficial/node-stratum-pool",
```

Comment:
re-add s-nomp here and below before creating the pull request to the original package